frontend/src/components/MinioArtifactPreview.test.tsx

```diff
@@ -1,3 +1,8 @@
+import MinioArtifactPreview from './MinioArtifactPreview';
```

Comment:
nit: Can you move imports under license declaration?

Reply:
ops. ide auto import hook not smart enuff to recognize license

Reply:
done